### DISCUSSION TITLE
OctoPlus.

### DISCUSSION THREAD

**Discussion by andyblac** [BottlecapDave/HomeAssistant-OctopusEnergy#485] [General]
Can anyone educate me on what this is....

<img width="857" alt="Screenshot 2023-10-25 at 10 25 02" src="https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/77936/e24e014c-2347-4eed-943d-9e0b257eb19c">
<img width="842" alt="Screenshot 2023-10-25 at 10 25 14" src="https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/77936/d2af6960-6095-4c41-b3e7-a70eb43100d2">

**Comment by BottlecapDave** [comment]
I think it's their new reward program. They had it last year with saving sessions - periods that you opt in, and depending on how much energy you reduce your consumption for during the same period you get rewarded points, which could be converted into credit. It looks like this expands upon that idea, but will allow you to get points in other ways (e.g. referrals) and potentially use them in ways other than credit (e.g. buy trees)

**Comment by andyblac** [comment]
ahh thanks.

**Comment by ITSpecialist111** [comment]
I'm just looking at this and the 'sensor.octopus_energy_saving_session_points' is saying I only have 1 point. However, on Octopoint I have around 500. Anyway of pulling this from the API and add to this integration?

**Reply by BottlecapDave** [reply]
As mentioned in https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/discussions/484#discussioncomment-7415416, octopoints are the successor to octopoints, so will need to be implemented separately.
